fastlane/lanes/localization.rb

```diff
@@ -120,10 +120,11 @@
       [:"play_store_screenshot_#{n}", File.join(metadata_folder, "screenshot_#{n}.txt")]
     end.to_h)
 
-    android_update_metadata_source(
-      po_file_path: File.join(metadata_folder, 'PlayStoreStrings.po'),
-      source_files: files,
-      release_version: options[:version]
+    update_po_file_for_metadata_localization(
```

Comment:
Created a local method to call `an_update_metadata_source` to avoid duplicating Git management. See bottom of the diff.